<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Season Scheduler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1121</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="calendar_page">
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>80</y>
           <width>431</width>
           <height>411</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCalendarWidget" name="calendarWidget">
            <property name="gridVisible">
             <bool>false</bool>
            </property>
            <property name="verticalHeaderFormat">
             <enum>QCalendarWidget::VerticalHeaderFormat::NoVerticalHeader</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="league_mgmt">
        <widget class="QListWidget" name="leagueListWidget">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>40</y>
           <width>271</width>
           <height>371</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLineEdit" name="createLeagueLine">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>430</y>
           <width>231</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Enter Team name</string>
         </property>
        </widget>
        <widget class="QPushButton" name="createLeagueButton">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>430</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Create League</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="leagues_page">
        <widget class="QListWidget" name="teamListWidget">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>90</y>
           <width>271</width>
           <height>561</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QTabWidget" name="league_tab">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>60</y>
           <width>482</width>
           <height>391</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="league_date_select">
          <attribute name="title">
           <string>Choose League Dates</string>
          </attribute>
          <widget class="QPushButton" name="leagueDateButton">
           <property name="geometry">
            <rect>
             <x>191</x>
             <y>330</y>
             <width>100</width>
             <height>29</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Set Times</string>
           </property>
          </widget>
          <widget class="QTableWidget" name="leagueDateTable">
           <property name="geometry">
            <rect>
             <x>13</x>
             <y>30</y>
             <width>450</width>
             <height>290</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="rowCount">
            <number>5</number>
           </property>
           <property name="columnCount">
            <number>7</number>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>64</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>64</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>53</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>53</number>
           </attribute>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row/>
           <row/>
           <column>
            <property name="text">
             <string>Sun</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Mon</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tues</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Wed</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Thurs</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Fri</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Sat</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="league_date_block">
          <attribute name="title">
           <string>Team Time Blocks</string>
          </attribute>
          <widget class="QLabel" name="weekBlockLable">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>10</y>
             <width>51</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTableWidget" name="timeBlockTable">
           <property name="geometry">
            <rect>
             <x>13</x>
             <y>30</y>
             <width>450</width>
             <height>290</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>64</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>64</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>53</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>53</number>
           </attribute>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignLeading|AlignTop</set>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignLeading|AlignTop</set>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignLeading|AlignTop</set>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignLeading|AlignTop</set>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignLeading|AlignTop</set>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Sun</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Mon</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tues</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Wed</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Thurs</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Fri</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Sat</string>
            </property>
           </column>
          </widget>
          <widget class="QPushButton" name="timeBlockButton">
           <property name="geometry">
            <rect>
             <x>191</x>
             <y>330</y>
             <width>93</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string>Set Times</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QLineEdit" name="addTeamLine">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="addTeamButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>60</y>
           <width>61</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Add Team</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="matchupsLine">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>510</y>
           <width>113</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="matchupsButton">
         <property name="geometry">
          <rect>
           <x>740</x>
           <y>510</y>
           <width>111</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Generate Matchups</string>
         </property>
        </widget>
        <widget class="QLabel" name="leagueLable">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>10</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="leagueStartButton">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>600</y>
           <width>201</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Set League Start Date</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="leaguesButton">
       <property name="text">
        <string>Leagues</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weekScheduleButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Week Schedule</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
